File: others/WorksApplication/Worksap Package/workspace/China_Intern/China_Intern_Java/src/jp/co/worksap/intern/entities/supermarket/SupermarketMstDTO.java
package jp.co.worksap.intern.entities.supermarket;

import jp.co.worksap.intern.entities.ICsvMasterDTO;

public class SupermarketMstDTO implements ICsvMasterDTO {
	/**
	 * 
	 */
	private static final long serialVersionUID = -3711231132225687846L;
	private Long supermarketId;
	private String address;
	private String tel;

	/**
	 * Default Constructor 
	 * 
	 * @param supermarketId
	 * @param supermarketName
	 * @param address
	 * @param tel
	 */
	public SupermarketMstDTO(Long supermarketId, String address,
			String tel) {
		super();
		this.supermarketId = supermarketId;
		this.address = address;
		this.tel = tel;
	}

	public Long getSupermarketId() {
		return supermarketId;
	}

	public String getAddress() {
		return address;
	}

	public String getTel() {
		return tel;
	}

}

File: others/WorksApplication/Worksap Package/workspace/China_Intern/China_Intern_Cpp/src/entities/supermarket/supermarket_mst_dto.cpp
#include "supermarket_mst_dto.h"
#include <iostream>
using namespace std;

SupermarketMstDTO::SupermarketMstDTO(long supermarket_id, string supermarket_name, string address,
		string tel) {
	this->supermarket_id = supermarket_id;
	this->supermarket_name = supermarket_name;
	this->address = address;
	this->tel = tel;
}

long &SupermarketMstDTO::get_supermarket_id() {
	return supermarket_id;
}

string &SupermarketMstDTO::get_supermarket_name() {
    return supermarket_name;
}

string &SupermarketMstDTO::get_address() {
	return address;
}

string &SupermarketMstDTO::get_tel() {
	return tel;
}

ostream &operator <<(ostream &out, SupermarketMstDTO &obj) {
	out << obj.supermarket_id << " " << obj.address << " "
			<< obj.tel;
	return out;
}

istream &operator >>(istream &in, SupermarketMstDTO &obj) {
	in >> obj.supermarket_id >> obj.address >> obj.tel;
	return in;
}

